src/components/Options/TimelineOptions.js

```diff
@@ -0,0 +1,16 @@
+const IsEnabledOptions = [
+  { value: "on", label: "ON" },
+  { value: "off", label: "OFF" },
+];
+
+const HasSyncedOptions = [
+  { value: "True", label: "TRUE" },
+  { value: "False", label: "FALSE" },
+];
+
+const InfiniteSessionsOptions = [
```

Comment:
Maybe this needs to be renamed to `SessionTypeOptions` with values `infinite`, `standard`. This way we will be easily add more options

Reply:
Done